### ISSUE TITLE
Discuss & Follow-through with Weapons Squad idea

### ISSUE THREAD

**Issue by derbismarck** [SignalCommunity/Wavelength#14]
Idea has come up to migrate our attachment teams system over to a Weapons squad system wherein a separate squad leader in charge of 6-12 guys acts as an addition squad in each platoon in order to render extra fire support. Discuss the idea in this feature request and give possible input into what it might look like in terms of composition.

**Comment by Winifridal** [comment]
Are we going to make each team 3 men or keep them at 2? I think we need to make them 3.

**Comment by derbismarck** [comment]
I don't think this is so good of an idea as I did before. Keeping it modular is better for our mission makers.

**Comment by Winifridal** [comment]
I like the idea but to be honest it's only really going to apply if we only have the players to slot 1 and a half squads, or the mission maker wants us to slot loads of attachment teams. My main issue though is that by putting them all in one squad and only giving them 343s you heavily limit their ability to move throughout the battlefield to where they need to be. 

TL;DR I like the idea of passing off command of weapon attachment teams to someone else but I don't think they should be treated in the same way a normal squad is.

**Comment by MysticalLatios** [comment]
The modular nature of the current system makes it easy for mission makers to do what they wish, maybe have just add a weapons squad lead or two so we can add stuff to them?

**Comment by Winifridal** [comment]
I believe the current idea is to have a WSL, and under him have any combination of asset teams (e.g 2x MMG + MAT or MMG + MAT + MTR) which could be predecided by the mission dev, or left up to the platoon leader. This doesn't change the modular potential in anyway.

**Comment by MysticalLatios** [comment]
Yeah we should just be adding a WSL or 2 then let mission makers add assets to them, we should not add in all the assets at start, they already exist as their two man teams.

**Comment by Winifridal** [comment]
Why or 2? We would only need one as it's unlikely we'll slot that many assets in the team anyway, and even if we do slot more than 3 their individual sizes mean that there probably won't even be 14 people in the squad total.

**Comment by KillPact** [comment]
Well since it seems that I've been quiet this entire time; this is the current idea floating around in my head, its a combination of what I personally wanted to do earlier, with some new things thrown in. I made a diagram. http://i.imgur.com/B2fPUwF.png The idea is to have another "SL" be in charge of the weapon elements directly. This is mirrored/inspired off IRL US inf structure. The weapons can be switched out by mission maker as they see fit. 

Why this idea is bouncing in my head is it organizes the weapons better, instead of being these small teams that are just lerking all about that usually just gets thrown into a squad anyways. It adds to their survivability with having a dedicated medic. Has one individual focused on coordination and radios, where one of the team members had to do before balance that with his other duties. 

If mission makers just make it a norm to have a weapons squad in missions with just MMGs in it, it allows us to fill that gap like we had during the Takistan mission where we had about 6 guys that had no where to go just because of attendance. MMGs is not that big of a force multiplier. When we dont have issues with numbers, we simply dont take weapons squad, easy as that. We actually did something practially identical to this on the Takistan mission, and it looked like it really sat well with everyone, and worked well, besides the obvious server issues that happened.

This wouldn't really require much change from the current framework, just have to adjust the radios, and maybe make a new mark? Anyways, thats the idea thats floating around in my head. Anyways, if you have questions, or want clarification to the idea, you know how to get ahold of me, lol.

P.S. I'm really tied while writing this, so grammer mistakes everywhere.

**Comment by Winifridal** [comment]
I said this to you but I want to try this out with the teams being on a Whiskey 343 net to see how that goes.

**Comment by KillPact** [comment]
What? anyone part of whiskey would already be on Chan4 343. The line of comms going out would be through the SL just like any other squad. The issue where I feel that you might be getting caught up with is this squad would not be any farther apart than a normal squad would be normally. Even less so cause they tend to be more static than the assaulting units.

**Comment by Winifridal** [comment]
It may improve their surviability but it greatly decreases their mobility and flexibility. It's not an issue if you have two MMG teams or two MAT teams but if you have a mix you may need them tasked on different things at anyone time which may mean that they need to move somewhere away from the squad.

**Comment by KillPact** [comment]
Thats the trade off, we often loose teams, and on "Death from Above" I lost majority of my HAT teams before they even got to the primary objective. This seems like a reoccurring theme. Do pitch an idea, and 3 man teams puts us back to where we were before. Lots of little teams, that die very easily, that clutter the radios, but are very flexible.

**Comment by KillPact** [comment]
eh, fuck the idea. Go with three men teams, and have them report to the PLT SGT.

**Comment by MysticalLatios** [comment]
The addition of a WSL would still allow mission makers to play with it easily if they wish.

**Comment by Winifridal** [comment]
What I am suggesting is do what you say and have a WSL in command of weapons teams, but replace their 343 with a 152 or something long range capable. Increase the team sizes to 3 as well.

I don't think the weapons squad should be viewed the same way a normal squad would. I think that deceases their potential effectiveness. If it's just MMG teams in whiskey (and mortar too) then it's not really an issue, but I think if you're restricting the movement of AT teams (i.e keeping them with WSL at all times) and not letting them be where they really need to be then you're putting the platoon at a disadvantage. It's definitely contextual but yeah I don't think Whiskey should necessarily be treated with the same mindset as a normal squad.

**Comment by Winifridal** [comment]
Regardless it's better than either clogging command net or passing them off to the PLT sergeant I think.

**Comment by KillPact** [comment]
Then the WSL makes no f'n sense. Just have them report to PLT SGT instead. Its essentially the same hat with a different name at that point. Anyways, that was my pitch, but ignore it now. You guys can go ahead and do whatever, I dont have the energy for all of this, if it was me, I'd just do the three man teams, have them report to PLT SGT (we dont have enough assets typically to overload him), and move on with it. We have been caught up with this for several days and we need to focus on other things.

**Comment by MysticalLatios** [comment]
Yeah having options for both would be nice, also 152s are configured to be on the same net as 117s and 148s so they are also an option for long range but not 117 range. I would like the WSL to have a 117 so they can be more independent then normal infantry, a hat team ran by WSL would be able to operate from the top of a mountain a click or two out. 

We don't have to force weapon teams into weapons squad in WL we can keep things as an option.

But at the same time PLT SGT already can do this roll. A medic would be nice though.

**Comment by Winifridal** [comment]
Eh I think this is a valid discussion to have.

If we do go the platoon sergeant route though then we need to have weapons channel for the 148 in the framework by default and have the assets be on that so we're not changing who they're commanded by from mission to mission.

**Comment by MysticalLatios** [comment]
We already have like 10 air nets lol, can remove all of them. Maybe they where already removed? I can do that also if needed its just some simple configs. I originally set them up as they are now.

That would give use space for a weapons net on long range without clogging things up.

**Comment by KillPact** [comment]
Here is my call then on it; have PLT SGT have a 148 and have a dedicated "Support Net". Have the leaders then of the support teams default to that net. When we run missions, we just need to take a PLT SGT everytime we have weapons on the field. I would suggest making teams 3 guys, but thats something harder to do FW wise and can wait for the future unless someone wants to do the work for it. Mean time, it can just be done as the missions are being developed.

**Comment by Winifridal** [comment]
I can sort out the 3 man teams as I've already done it myself

**Comment by derbismarck** [comment]
My view: The way KP outlined the system in his initial post is perfect. That is what I want.

Realistically: In order to do this we would need to completely restructure the mission.sqm again. This is a process that takes upwards of ten hours of highly intensive complex work (thanks to BI not having a fucking tool to do it or something in the game that allows you to do it) where any single mistake will break the entire sqm. As such, I only trust myself or JonBons (and maybe MysticalPony) to make these changes. The fact of the matter is that I simply don't have the amount of motivation required to make this happen, so unless Mystical or JonBons want to take a lot of time out of their busy schedules to do something which only has some little bits of extra stuff (lots of dev work for not much change) then it won't be happening any time soon.
